packages/gatsby/cache-dir/default-html.js

```diff
@@ -15,9 +15,6 @@ export default function HTML(props) {
       </head>
       <body {...props.bodyAttributes}>
         {props.preBodyComponents}
-        <noscript key="noscript" id="gatsby-noscript">
-          This app works best with JavaScript enabled.
-        </noscript>
```

Comment:
should this change be part of this PR?

Reply:
Also for reference - this is PR that added this initially https://github.com/gatsbyjs/gatsby/pull/10945 so worth checking reasoning and see if we don't break that (or at least it's conscious decision)